elements/pl_file_editor/pl_file_editor.py

```diff
@@ -2,10 +2,17 @@
 import lxml.html
 import chevron
 import base64
+import hashlib
 
 
 def get_answer_name(file_name):
-    return '_file_editor_{0}'.format(file_name)
+    return '_file_editor_{0}'.format(hashlib.sha1(file_name.encode('utf-8')).hexdigest())
```

Comment:
This is just a thought rather than an immediate change, but I wonder if we are really gaining anything here above just using the filename with any non-printing characters converted to underscores or something like that. We should be checking for name collisions in any case. Using a more human-readable answer name would make looking at the raw data more pleasant.